README.md

```diff
@@ -42,20 +42,26 @@ Visit her profile and send her some love for her amazing job!
 
 ## Getting Started
 
+To watch the video showcasing the process of building this site please go to this Youtube link:
+
+- [`https://www.youtube.com/watch?v=lG3Q4bRfKuQ`](https://www.youtube.com/watch?v=lG3Q4bRfKuQ)
+
+If you want to visit the Figma Design System that I created to help me building this project you can visit the following link
+
+- [`https://www.figma.com/file/uxDIr6zq8lVsdJH08j6b0w/capstone-1`](https://www.figma.com/file/uxDIr6zq8lVsdJH08j6b0w/capstone-1)
+
 If you want to visit the deployed app just visit the following link:
 
-- [`https://metsanpeitto.github.io/Portfolio_1/`](https://metsanpeitto.github.io/Portfolio_1/)
+- [`https://metsanpeitto.github.io/capstone-1/index.html`](https://metsanpeitto.github.io/capstone-1/index.html)
 
 If you want a copy of this file go to the github repository and download it
 
-- `git clone https://github.com/Metsanpeitto/Portfolio_1`
+- [`git clone https://github.com/Metsanpeitto/capstone-1`](https://github.com/Metsanpeitto/capstone-1)
 
 After cloning it you can see the project opening the index.html file with your favorite browser ( Firefox for example )
```

Comment:
- [ ] Good job here, your Readme is very descriptive and engaging 👏🏾 There are a few tweaks you do have to make and you're good to go:
- Kindly add clearer instruction on how to clone and run your project locally, keeping in mind that there might be non-technical users trying to use it as well. Make the steps necessary to run it locally detailed and easy to follow.

Reply:
Thanks for your time Temmarie. I really appreciate it. 

 Thanks for pointing out the problem with the image size. I used different end for this project but didn't create a separate media query for tablets (from 600px to 1024px). 

I will fix that now.

However I have to defend the reason why I copy the content and decided to don't customize nothing.

I had a lot of doubts about what really was expected from us. Everyone seemed to have different opinions, so I asked to Delice, who I think is the leader of the code reviewers, I did it on Friday last week when I started working in this project. 
Here you can see the full message :   

             
"  Abraham_Rodriguez_Martinez  10:20 AM
Hi Delice!
My name is Abraham Rodriguez and I am a student in Microverse 2.0, currently heading towards the first capstone project. 
I have some doubts and after asking my mentor, other students and reading many times through the Capstone requirements I still have some questions that I don't know how to answer, and thought that maybe you could help me with them. 
If I should redirect this kind of question to another source, please let me know and please save your time by omitting the rest of this message.
I would like to know for sure, what is the level of customization that I can strive for.
In the documentation I read : 
" We provide some design guidelines for you to create the website, but you must personalize the content…"
As well as this other one : 
"You should personalize the content of your page. Choose a topic first and then design your webpage, keeping your topic in mind while you design it so that your page design works well for the topic you choose."
However, some lines below I read : 
"You should follow these design guidelines (https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), including:
  - Colors.
  - Typography: font face, size and weight.
  - Layout: composition and space between elements."
So I am confused. Can I create my own design system and build a page from scratch that includes a 'landing page' and an 'about section'? 
Or, Should I replicate that page 'pixel perfect' and take the freedom to only modify the pictures and the text?
I have experience as Ui/Ux designer and have my own Design Systems, as well as some experience with threejs that I would like to apply having the chance to. 
If the goal of this task is to recreate a pixel perfect replica of the given site I will do it, however If I can apply all my experience and try to come out with the best product I can, I would love to take the chance.
Hope the questions were clear and I didn't waste your time. 
Thanks beforehand.
Abraham Rodriguez
Behance
CC Global Summit 2015
The Global Summit brings together the community of experts, academics, and activists who comprise the Creative Commons affiliate network in a different country every two years. I conceptualized and designed websites of the event that will be held in Korea&#8230; (Not automatically expanded because 3 MB is too large. You can expand it anyway or open it in a new window.)
Delice Lydia  10:33 AM
Hello @Abraham_Rodriguez_Martinez,
Thanks to reach out i understand your concern and the reason behind capstone
is applying what you learned in the whole course. Take this as learning first of all you might
have some background knowledge yes but when you decided to join MICROVERSE is because
you want to reach somewhere else right?
So for the capstone yes they chose the design we need to follow but they allow to come up with
a new idea while working on your project, to think in a real world .. you are free to personalize your
content but still following the design.
Take this as learning because you will have forever to design your project as how you want
and keep your head up!
Thanks
7:45
Abraham_Rodriguez_Martinez  10:35 AM
Thanks for answering Delice. I think I got it know.
I will replicate it pixel perfect. I hope it is ok.
Have a nice weekend and thanks for your time. "

You can see how I ended the message with the line "I will replicate it pixel perfect. I hope it is ok."

And her answer was affirmative, so I proceeded to take this project as a proffesional one, where I am commanded to replicate a site. I knew that different code reviewers would tell me different things, so I went to the code review leader and asked to her directly.
 That was my thought process and the reason why I didn't customized anything and why I think not any customization is needed. I hope you can understand my effort and will to do what is expected from me. I took the time to inform myself about the requirements and what they meant, asking to the person who would have the last word in this case.

Reply:
Hi @Metsanpeitto, thanks for your feedback. Firstly, Delice is not the head of the TSEs. But it's a good thing you went to her concerning this issue before embarking on the project. I think there has been a bit of a misunderstanding in communication. She gave an affirmative to take the project as a learning process and still mentioned personalising the content. It could be that at the end, when you said it will be replicated pixel perfect, she assumed you meant replicating the design perfectly but not the content. I appreciate you taking out the time to reach out to me concerning this issue.
From the instruction, it says to follow the design but personalise or customise the content. So, I will reach out to one of the TSE leaders and ask for her opinion on this and get back to you. For now, kindly fix the issue with the page responsiveness.

Reply:
Hello @Metsanpeitto, 
it is good to see how far you reached with your capstone but i wanted to make
clear what I said, if you can read well my message I said that you should personalize your content
but still follow the given design 

![ls](https://user-images.githubusercontent.com/44745855/124263395-7ce30580-db33-11eb-8edc-7cee38c214c4.PNG)

I apologize for not explaining clearly.

Thank you!

Reply:
Thanks, Tamara, I have already fixed the responsiveness problem and added more detailed instructions on how to download and explore the project for a nontechnical person as you suggested.
Please, let me know who is the code review leader, I would love to have access to her/ him in the future. Sometimes I have questions that no my peers or my mentor know how to answer and I feel quite impotent without knowing who could guide me.

I hope that you don't take my position as defying, anything negative, or even personal.
You are kind by exposing your point and taking the time to ask other people among the code reviewers to give me the right answer. I really appreciate that.

 I just wanted to do it as well as I could. You can see that I started 3 days before the actual date working on the project and I have spent every waking hour on it, to achieve a level that would be expected in a professional environment. 
Going back now to modify the content would add a lot of extra work and I don't really understand the purpose of it. Maybe it would showcase my abilities to create 'copyright content'? Please let me know the benefits it would bring to do so.
My point is that I would like to use this site in my actual portfolio (https://portfolio.waldenberginc.com) and replicating it as it is, along with the Figma DesignSystem would showcase better my abilities as a designer and as a developer. 
Changing the content would add a layer of confusion to the potential employers who would take a look at it.

If changing the content is mandatory and really relevant, please, let me know the constraints I should consider and what is the detail expected from me. I am more and more confused every time I see the different opinions that code reviewers gave to my peers and would like to end the project today Friday so I can leave this chair for some hours after a whole week working only on this project, rest a little bit and recharge before the next module starts.

Thanks again for your time, your kind words, and I wish you have a nice weekend.
A pleasure talking to you.
Abraham Rodriguez

Reply:
Hi @Metsanpeitto , thanks for making the required changes. I understand how hard it must be to have to make changes to a project that you've already worked really hard on to achieve mastery. Here's what I suggest, since you'd like to put this actual project and the first reviewer didn't mention it, I'll mark it as optional, so you don't have to change the content.

Reply:
Thanks for replying Delice. 

However, I still don't understand why the sentence '...you are free to personalize your content' it's supposed to mean 'You 'have' to personalize your content'.

 If I read that, I still understand that I can 'do it' or 'don't do it', because that is the meaning of the word free. 
Quote from Wikipedia:
'' _Freedom, generally, is having the ability to act or change without constraint. Something is "free" if it can change easily and is not constrained in its present state._ ''
If I have to mandatorily create my own content that is a constraint, so the word free used in that context has a misleading effect and generates confusion.

If the really relevant point here is to obey what my superior says without questioning the meaning of their words, please let me know and I will get back to code and end the project in the terms desired by the ones who decide and qualify my work. Also, it would help me to understand in the future, what is expected from us, and what are the more desired traits that code reviewers want on us. 

I would love it if you can understand my position as well as what my will was with this project and with these messages. I only wanted to do as best as I can and have clear what is expected from me.

Thanks for your time and I hope you don't misunderstand me.

Reply:
Thanks, Tamara. 

I hope my words were not misunderstood. I try to never say or do anything that could be taken as unpolite because that is never my intention. However, this medium of communication lacks the human aspect that would help to don't mistake our attitudes when trying to make our points. So, please, forgive that.
In the future, I will do all the needed questions and require clarifications over the points I find confusing before I start any project so I have a more clear understanding of the requirements and I save myself and your team our precious time.

 Thanks for giving me the chance of submitting the project as it is now and for understanding my purpose of using the project in my portfolio with the original content. Also thanks for the link for the TSE.

Have a nice weekend and thanks for your patience.

Reply:
You're welcome Abraham, your request and responses were not in any way impolite. I'm glad to have been able to help with this issue. You've done an astounding job in this capstone project. So I hope It gets approved on the next submission. All the best!